### ISSUE TITLE
obs4MIPs_CV.json file lacks a number of fields found in the obs4MIPs_source_id.json file.

### ISSUE THREAD

**Issue by JimBiardCics** [PCMDI/obs4MIPs-cmor-tables#80]
The (currently only) source_id entry for REMSS-PRW-6-6-0 in the obs4MIPs_CV.json file lacks the fields listed below.

- description
- institution_id
- label
- source_id

**Comment by durack1** [comment]
@JimBiardCics are you sure you're pointing at the latest?

[PCMDI/obs4MIPs-cmor-tables/obs4MIPs_source_id.json](https://github.com/PCMDI/obs4MIPs-cmor-tables/blob/master/obs4MIPs_source_id.json)

```
    "source_id":{
        "REMSS-PRW-6-6-0":{
            "description":"Water Vapor Path",
            "institution_id":"RSS",
            "label":"REMSS PRW 6.6.0",
            "region":"global",
            "release_year":"2017",
            "source_id":"REMSS-PRW-6-6-0",
            "source_label":"REMSS-PRW",
            "source_type":"satellite_blended"
        }
}
```
I'll assume there is some confusion with this issue and will close, reopen if you have more info to add

**Comment by JimBiardCics** [comment]
@durack1 @dnadeau4 Notice that I was saying that the CV file lacks the fields. I wasn't sure if this was OK or not, so I made the issue.

I since discovered that having all the fields from the source_id file in the CV file can cause problems, and that the 'source' element doesn't get created from the CV file contents, so I'm guessing the contents of the source_id file are not just copied into the CV file. Which is fine. Just wanted to be sure.
